### ISSUE TITLE
Missing plugin VPR docs between placeholder generation and last commit date.

### ISSUE THREAD

**Issue by mashhurs** [elastic/docs-tools#100]
### Description
We generated and added `logstash-integration-logstash` plugin placeholder to the repo. After that, when we generated VPR we witnessed that workflow couldn't catch es-output changes. The reason behind, when we generated VPR, we get the last commit date (`git log -1 --date=short --pretty=format:%cd`) and seek the new published gems after that date.

### Solutions
1) place a caution to encourage running VPR before on placeholder generator
2) when generating placeholder, also catch the VPR changes
3) add an ability to generate VPR docs for individual plugin

2) is better from operation point of view.

**Comment by karenzone** [comment]
@mashhurs, we'll soon be adding the snmp integration.  If I'm reading the root cause and proposal correctly, Option 1 provides an easy work-around. That is, before I use the github action to generate the placeholder page, I just need to generate the VPR docs, right?

**Comment by mashhurs** [comment]
> @mashhurs, we'll soon be adding the snmp integration. If I'm reading the root cause and proposal correctly, Option 1 provides an easy work-around. That is, before I use the github action to generate the placeholder page, I just need to generate the VPR docs, right?

That's right as I remember. And please share with me if you face any issues. This is an opportunity to test/validate our latest approach. Thank you remembering this :)
